common-content-dataflow/modules/ROOT/partials/changelog.adoc

```diff
@@ -0,0 +1,101 @@
+= Changelog
+
+This page lists changes to the Dataflow Flex Template for BigQuery and/or Google Cloud to Neo4j.
+
+You can check the deployment status of a release by running `gsutil ls gs://dataflow-templates/`.
+
+Releases are gradually deployed, some regions may be updated before others.
+To check the release status of https://cloud.google.com/compute/docs/regions-zones[a particular region], you can run `gsutil ls gs://dataflow-templates-$\{region}/` (e.g. `gsutil ls gs://dataflow-templates-europe-west2/`).
+
```

Comment:
Shall we also mention that it is not possible to run any version other than _latest_?